scripts/exrrfs_ioda_bufr.sh

```diff
@@ -26,6 +26,22 @@ yaml_list=(
 #"prepbufr_vadwnd.yaml"
 )
 
+# Copy empty ioda file to data/obs.
+# Use these as the default when bufr2ioda doesn't create a ioda.
+# Otherwise JEDI will crash due to missing ioda file
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_adpsfc.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_adpupa.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_aircar.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_aircft.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_ascatw.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_gpsipw.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_msonet.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_proflr.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_rassda.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_satwnd.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_sfcshp.nc
+${cpreq} ${PARMrrfs}/ioda_empty.nc ioda_vadwnd.nc
```

Comment:
As @MatthewPyle-NOAA mentioned, we need to put these fixed ioda_empty.nc files into ${FIXrrfs}

You can change `${PARMrrfs}` to `${FIXrrfs}/jedi`.  

Also, share me the ioda_empty.nc file and I will take care of the rest part.

Reply:
@guoqing-noaa, I appreciate you offering to handle putting the empty ioda file in fix location. I currently have a copy of it on Hera here:
`/scratch2/NCEPDEV/fv3-cam/Donald.E.Lippi/RRFSv2/workflow/v2.0.9.5/parm/ioda_empty.nc`

Reply:
I staged it into FIX_RRFS2. 

You may do as follows to add it to the workflow repo:
```
cd ${HOMErrfs}/fix/jedi
cp -rp /scratch1/BMC/wrfruc/gge/rrfs-workflow/fix/jedi/ioda_empty.nc .   # need -rp as it is a link in the repo
git add ioda_empty.nc
git commit -m ‘add ioda_empty.nc into FIXrrfs'
```

Reply:
I think I got it. Thank you very much for those instructions!